### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by alexjfisher**

### CODE REVIEW

File: spec/unit/provider/network_config/redhat_spec.rb [hunk 1]
```diff
@@ -25,10 +25,10 @@ def fixture_data(file)
   end
 
   describe 'selecting files to parse' do
-    let(:network_scripts_path) { fixture_file('network-scripts') }
-
     subject { described_class.target_files(network_scripts_path).map { |file| File.basename(file) } }
 
+    let(:network_scripts_path) { fixture_file('network-scripts') }
```
Comment (igalic) [inline spec/unit/provider/network_config/redhat_spec.rb]: i wasn't bold enough to do this…
i was too afraid…
rspec paralyses me sometimes :(
(and sometimes i spend 2 hours hunting down a bug in pry/byebug)

Comment (alexjfisher) [inline spec/unit/provider/network_config/redhat_spec.rb]: You were too afraid, but I may have been too stupid?

Comment (alexjfisher) [inline spec/unit/provider/network_config/redhat_spec.rb]: https://www.relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let
> Note that let is lazy-evaluated: it is not evaluated until the first time
the method it defines is invoked.

I'm going to pretend that means my change is *definately* ok.

Comment (igalic) [inline spec/unit/provider/network_config/redhat_spec.rb]: can something also be forward lazy???

**Pull request merged**